routes/index.js

```diff
@@ -0,0 +1,22 @@
+// 1. IMPORTACIONES
+
+const express    = require("express");
+const res        = require("express/lib/response");
+const router     = express.Router();
+
+//2. RUTEO
+router.get("/index", (req,res) =>{
```

Comment:
Te recomiendo tener un ruta raiz, esta se va a renderizar cuando estes en el puerto sin agregar nngun parametro en la search bar solo el puerto es como su nombre indica la raiz del server en este caso podemos dejar index:
```suggestion
router.get("/", (req,res) =>{
```